src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,16 +19,15 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
-    private string $routeName = '';
-    private array $missingParameters = [];
```

Comment:
I think this class must be reverted. We loose the default value on the property now (CPP only defines the default value on the constructor parameter, not the property).

This means that classes extending this and not calling `parent::__construct()` will get an error when calling `getRouteName()` whereas they would get an empty string with the existing code.